tmt/steps/discover/fmf.py

```diff
@@ -557,6 +557,11 @@ def assert_git_url(plan_name: Optional[str] = None) -> None:
                 'modified-ref',
                 tmt.utils.default_branch(repository=self.testdir, logger=self._logger))
             self.info('modified-ref', modified_ref, 'green')
+            ref_commit = self.run(
+                Command('git', 'rev-parse', '--short', f"{modified_ref}"),
+                cwd=self.testdir)
+            assert ref_commit.stdout is not None
+            self.debug(f"{modified_ref}: {ref_commit.stdout.strip()}")
```

Comment:
I *think* `debug` would do the key/value formatting for you, `debug(modified_ref, ref_commit.stdout.strip())`.

Reply:
fixed in 8fb02327. not sure about if I should use color or not

Reply:
No idea. I wouldn't, for debugging I felt it rarely helped me. Your call.